server/services/mi-light/lib/light/light.configureBridge.js

```diff
@@ -0,0 +1,66 @@
+const logger = require('../../../../utils/logger');
+const { NotFoundError } = require('../../../../utils/coreErrors');
+const {
+  BRIDGE_EXTERNAL_ID_BASE,
+  BRIDGE_MODEL,
+  BRIDGE_IP_ADDRESS,
+  BRIDGE_MAC,
+  BRIDGE_NAME,
+  BRIDGE_TYPE,
+} = require('../utils/consts');
+
+/**
+ * @description Configure the milight bridge.
+ * @param {string} Mac - Mac Address of the Mi-Light Bridge.
+ * @example
+ * configureBridge('F0:FE:6B:A6:03:EB');
+ */
+async function configureBridge(Mac) {
```

Comment:
We use camelCase to name variables, so "Mac" should be "mac".

I'm surprised that ESLint is not checking this... I'm going to add this to eslint !